<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QApplication</class>
 <widget class="QMainWindow" name="QApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="SwitcherWindow">
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="noticeTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>521</width>
      <height>121</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>521</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="button_jdk9">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>280</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_jdk9">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>240</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OracleJDK 9</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_jdk8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_jdk8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OracleJDK 8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_jdk7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_jdk7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OracleJDK 7</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>461</width>
       <height>111</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>9</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans Mono'; font-size:9pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:normal;&quot;&gt;Apart from the base code, Oracle JDK includes, Oracleâ€™s implementation of Java Plugin and Java WebStart. Also includes third party closed source and open source components like graphics rasterizer and Rhino respectively.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="oracle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>oracle.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>download.oracle.com</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>line</zorder>
    <zorder>textBrowser</zorder>
    <zorder>oracle</zorder>
    <zorder>button_jdk9</zorder>
    <zorder>label_jdk8</zorder>
    <zorder>label_jdk9</zorder>
    <zorder>button_jdk7</zorder>
    <zorder>button_jdk8</zorder>
    <zorder>label_jdk7</zorder>
    <zorder>label</zorder>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>501</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_openjdk7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OpenJDK 7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_openjdk7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_openjdk8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>280</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_openjdk8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OpenJDK 8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>461</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans Mono'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'DejaVu Sans'; color:#ffffff;&quot;&gt;OpenJDK Font Renderer and Oracle JDK Flight Recorder are the noticeable major difference between Oracle JDK and OpenJDK&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'DejaVu Sans'; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'DejaVu Sans'; color:#ffffff;&quot;&gt;OpenJDK is completely free and can be used in accordance with GPL v2 license&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="openjdk">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>openjdk.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>openjdk.java.net/install</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>line_2</zorder>
    <zorder>textBrowser_2</zorder>
    <zorder>openjdk</zorder>
    <zorder>label_openjdk7</zorder>
    <zorder>button_openjdk7</zorder>
    <zorder>label_openjdk8</zorder>
    <zorder>button_openjdk8</zorder>
    <zorder>label_2</zorder>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>450</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Roboto'; font-size:8pt;&quot;&gt;Archlinux-Versionswitcher v1.0.1&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Roboto'; font-size:8pt;&quot;&gt;https://github.com/xNNism/Archlinux-Versionswitcher&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Roboto'; font-size:8pt;&quot;&gt;2017 - xNN | Blackbyte&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Roboto'; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>noticeTextEdit</zorder>
   <zorder>button_exit</zorder>
   <zorder>textBrowser_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
